### PR TITLE
fix: Add error message to device deletion (WEBAPP-6484)

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: src/script/view_model/ActionsViewModel.js [hunk 1]
```diff
@@ -112,23 +114,33 @@ export class ActionsViewModel {
...
+          action: async password => {
+            if (!isSending) {
+              isSending = true;
+              try {
+                await this.clientRepository.deleteClient(clientEntity.id, password);
+                this.modalsViewModel.hide();
+                resolve();
+              } catch (error) {
+                this.modalsViewModel.errorMessage(expectedErrors[error.label] || error.message);
+                isSending = false;
```
Comment (Yserz) [inline src/script/view_model/ActionsViewModel.js:142]: finally {isSending = false}

File: src/page/template/modals.htm [hunk 2]
```diff
@@ -20,15 +20,18 @@
...
         </form>
       <!-- /ko -->
       <!-- ko if: hasInput() -->
-        <form data-bind="submit: () => doAction(confirm)">
+        <form data-bind="submit: () => doAction(confirm, content().closeOnConfirm)">
           <input maxlength="64" class="modal__input" data-bind="textInput: inputValue, attr: {placeholder: content().inputPlaceholder}, hasFocus: inputFocus"/>
         </form>
       <!-- /ko -->
+      <!-- ko if: errorMessage() -->
+      <div class="modal__input__error" data-bind="text: errorMessage"></div>
```
Comment (ffflorian) [inline src/page/template/modals.htm]:
```suggestion
        <div class="modal__input__error" data-bind="text: errorMessage"></div>
```

File: src/script/view_model/ActionsViewModel.js (hunk 1 above)
Comment (AndyLnd) [reply src/script/view_model/ActionsViewModel.js:142]: we really don't care about isSending anymore once this is successful, as we then leave the scope. if the method errors, however, you should be able to send the request again.

Comment (ffflorian) [reply src/script/view_model/ActionsViewModel.js:142]: But @Yserz is right, it also makes sense when reading it.

Comment (Yserz) [reply src/script/view_model/ActionsViewModel.js:142]: It's true, technically this doesn't change anything. But for a reader it's more clear what should happen.

Comment (ffflorian) [inline src/script/view_model/ActionsViewModel.js:142]:
```suggestion
              } finally {
                isSending = false;
```

**Pull request merged**